plugins/org.eclipse.emf.mwe2.language/META-INF/MANIFEST.MF

```diff
@@ -12,7 +12,7 @@ Require-Bundle: org.eclipse.emf.ecore,
  org.eclipse.equinox.common;bundle-version="3.6.0",
  com.google.inject;bundle-version="3.0.0",
  org.eclipse.xtext;bundle-version="2.14.0",
- org.eclipse.xtext.common.types;bundle-version="2.14.0";visibility:=reexport,
+ org.eclipse.xtext.common.types;bundle-version="2.14.0",
```

Comment:
This appears to be a regression with the mergable manifest

Reply:
can you open a separate issue for that?

Reply:
is this a manual change?